analyses/transcriptomic-dimension-reduction/README.md

```diff
@@ -0,0 +1,130 @@
+# Dimension reduction: gene expression data
+
+This module runs dimension reduction—namely PCA, UMAP, and t-SNE—on gene expression data from kallisto and RSEM.
+
+To the run the whole pipeline and produce 4 plots with panels for all three approaches
```

Comment:
What are the three approaches being referred to? This itemized list doesn't make it clear to me.

Reply:
PCA, UMAP, t-SNE